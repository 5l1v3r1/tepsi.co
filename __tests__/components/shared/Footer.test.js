import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../../src/components/shared/Footer';

it('renders correctly', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});
