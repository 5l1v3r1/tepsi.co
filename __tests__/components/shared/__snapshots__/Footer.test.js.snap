exports[`test renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <footer
    className="Footer">
    <nav
        className="Footer-menu">
        <a
            className="Footer-menu__item"
            href="https://github.com/alpcanaydin/tepsi.co"
            rel="noopener noreferrer"
            target="_blank">
            kaynağı görüntüle
        </a>
        <a
            className="Footer-menu__item"
            href="https://twitter.com/alpcanaydin"
            rel="noopener noreferrer"
            target="_blank">
            @alpcanaydin
        </a>
        <a
            className="Footer-menu__item"
            href="https://twitter.com/pinquitte"
            rel="noopener noreferrer"
            target="_blank">
            @pinquitte
        </a>
        <a
            className="Footer-menu__item"
            href="https://twitter.com/bigrandma"
            rel="noopener noreferrer"
            target="_blank">
            @bigrandma
        </a>
        <a
            className="Footer-menu__item"
            href="https://twitter.com/batuhankrskl"
            rel="noopener noreferrer"
            target="_blank">
            @batuhankrskl
        </a>
        <a
            className="Footer-menu__item"
            href="https://twitter.com/hsnaydd"
            rel="noopener noreferrer"
            target="_blank">
            @hsnaydd
        </a>
    </nav>
</footer>,
  "nodes": Array [
    <footer
      className="Footer">
      <nav
            className="Footer-menu">
            <a
                  className="Footer-menu__item"
                  href="https://github.com/alpcanaydin/tepsi.co"
                  rel="noopener noreferrer"
                  target="_blank">
                  kaynağı görüntüle
            </a>
            <a
                  className="Footer-menu__item"
                  href="https://twitter.com/alpcanaydin"
                  rel="noopener noreferrer"
                  target="_blank">
                  @alpcanaydin
            </a>
            <a
                  className="Footer-menu__item"
                  href="https://twitter.com/pinquitte"
                  rel="noopener noreferrer"
                  target="_blank">
                  @pinquitte
            </a>
            <a
                  className="Footer-menu__item"
                  href="https://twitter.com/bigrandma"
                  rel="noopener noreferrer"
                  target="_blank">
                  @bigrandma
            </a>
            <a
                  className="Footer-menu__item"
                  href="https://twitter.com/batuhankrskl"
                  rel="noopener noreferrer"
                  target="_blank">
                  @batuhankrskl
            </a>
            <a
                  className="Footer-menu__item"
                  href="https://twitter.com/hsnaydd"
                  rel="noopener noreferrer"
                  target="_blank">
                  @hsnaydd
            </a>
      </nav>
</footer>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Footer />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <footer
          className="Footer">
          <nav
                    className="Footer-menu">
                    <a
                              className="Footer-menu__item"
                              href="https://github.com/alpcanaydin/tepsi.co"
                              rel="noopener noreferrer"
                              target="_blank">
                              kaynağı görüntüle
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/alpcanaydin"
                              rel="noopener noreferrer"
                              target="_blank">
                              @alpcanaydin
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/pinquitte"
                              rel="noopener noreferrer"
                              target="_blank">
                              @pinquitte
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/bigrandma"
                              rel="noopener noreferrer"
                              target="_blank">
                              @bigrandma
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/batuhankrskl"
                              rel="noopener noreferrer"
                              target="_blank">
                              @batuhankrskl
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/hsnaydd"
                              rel="noopener noreferrer"
                              target="_blank">
                              @hsnaydd
                    </a>
          </nav>
</footer>,
        "_debugID": 2,
        "_renderedOutput": <footer
          className="Footer">
          <nav
                    className="Footer-menu">
                    <a
                              className="Footer-menu__item"
                              href="https://github.com/alpcanaydin/tepsi.co"
                              rel="noopener noreferrer"
                              target="_blank">
                              kaynağı görüntüle
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/alpcanaydin"
                              rel="noopener noreferrer"
                              target="_blank">
                              @alpcanaydin
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/pinquitte"
                              rel="noopener noreferrer"
                              target="_blank">
                              @pinquitte
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/bigrandma"
                              rel="noopener noreferrer"
                              target="_blank">
                              @bigrandma
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/batuhankrskl"
                              rel="noopener noreferrer"
                              target="_blank">
                              @batuhankrskl
                    </a>
                    <a
                              className="Footer-menu__item"
                              href="https://twitter.com/hsnaydd"
                              rel="noopener noreferrer"
                              target="_blank">
                              @hsnaydd
                    </a>
          </nav>
</footer>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Footer />,
}
`;
